src/views/v2/commerce/checkout/cart/item/sub-total.php

```diff
@@ -28,5 +28,5 @@
 
 ?>
 <div class="tribe-tickets__commerce-checkout-cart-item-subtotal">
-	<?php echo esc_html( tribe_format_currency( $item['sub_total'] ) ); ?>
+	<?php echo wp_kses_post( \TEC\Tickets\Commerce\Utils\Price::to_currency( $item['sub_total'] ) ); ?>
```

Comment:
We need to set the @version of this template TBD and add a @since TBD with a small description of the changes.